static/templates/recent_topics_list.hbs

```diff
@@ -0,0 +1,25 @@
+<div id="recent_topics_filter_buttons" class="btn-group" role="group" aria-label="Recent Topics">
+    <button data-filter="all" type="button" class="btn btn-default btn-recent-filters">All</button>
+    <button data-filter="unread" type="button" class="btn btn-default btn-recent-filters">Unread</button>
+    <button data-filter="bookmarked" type="button" class="btn btn-default btn-recent-filters">Bookmarked</button>
+    <button data-filter="participated" type="button" class="btn btn-default btn-recent-filters">Participated</button>
+    <input type="text" id="recent_topics_search" placeholder="Search Stream / Topics names" aria-label="search-topics">
+</div>
+    <div class="tableFixHead">
+        <table class="recents_table table table-striped table-bordered table-responsive table-hover">
+            <thead class="thead-dark">
+                <tr>
+                    <th>Unread</th>
+                    <th>Stream</th>
+                    <th>Topic</th>
+                    <th>Actions</th>
+                    <th>Active Users</th>
+                    <th>Last message</th>
```

Comment:
Tag these strings for i18n.  Also, that will cause you to notice these will fail our capitalization linter; should be "Active users" (we always use "Sentence case" in our labels)